logger/tests/integration_tests.rs

```diff
@@ -323,64 +269,47 @@ async fn generate_and_stream_logs() {
+            data: format!("TRACE span_name2{{deployment_id=\"{DEPLOYMENT_ID}\"}}: integration_tests: inside span 2 event\n")
         },
     );
 }
 
 /// For the service logs the deployment id will be retrieved from the spans of functions
 /// instrumented with the deployment_id field, this way we can choose which spans we want
 /// to associate with a deployment and record in the logger.
-fn generate_service_logs(port: u16, deployment_id: String, generator: fn(String)) {
-    generate_logs(
-        port,
-        deployment_id,
-        generator,
-        vec![KeyValue::new("service.name", "test")],
-    );
+fn generate_service_logs(
+    client: LoggerClient<tonic::transport::Channel>,
+    deployment_id: String,
+    generator: fn(String),
+) {
+    generate_tracing_logs(client, deployment_id, generator);
 }
 
 /// For the shuttle-runtime logs we want to add the deployment id to the top level attributes,
 /// this way we can associate any logs coming from a runtime with a deployment.
-fn generate_runtime_logs(port: u16, deployment_id: String, generator: fn(String)) {
-    generate_logs(
-        port,
-        deployment_id.clone(),
-        generator,
-        vec![
-            KeyValue::new("service.name", "test"),
-            KeyValue::new("deployment_id", deployment_id),
-        ],
-    );
+fn generate_runtime_logs(
+    client: LoggerClient<tonic::transport::Channel>,
+    deployment_id: String,
+    generator: fn(String),
+) {
+    generate_tracing_logs(client, deployment_id, generator);
 }
 
 /// Helper function to setup a tracing subscriber and run an instrumented fn to produce logs.
-fn generate_logs(
-    port: u16,
+fn generate_tracing_logs(
+    client: LoggerClient<tonic::transport::Channel>,
     deployment_id: String,
     generator: fn(String),
-    resources: Vec<KeyValue>,
 ) {
-    // Set up tracing subscriber connected to the logger server.
-    let tracer = opentelemetry_otlp::new_pipeline()
-        .tracing()
-        .with_exporter(
-            opentelemetry_otlp::new_exporter()
-                .tonic()
-                .with_endpoint(format!("http://127.0.0.1:{port}")),
-        )
-        .with_trace_config(
-            opentelemetry::sdk::trace::config()
-                .with_resource(opentelemetry::sdk::Resource::new(resources)),
-        )
-        .install_simple()
-        .unwrap();
-    let otel_layer = tracing_opentelemetry::layer().with_tracer(tracer);
-
-    let _guard = tracing_subscriber::registry()
-        .with(otel_layer)
-        .set_default();
+    // Set up tracing subscriber connected to the logger server
+    let layer = tracing_subscriber::fmt::layer()
+        .with_writer(Mutex::new(LoggerLayer::new(
+            deployment_id.clone(),
+            SHUTTLE_SERVICE.into(),
+            client,
+        )))
+        .with_ansi(false);
+    let _guard = tracing_subscriber::registry().with(layer).set_default();
```

Comment:
I'm not sure setting up tracing for these tests still makes sense. We should now be okay will calling the `StoreLogs` method directly and can have fever tests

Reply:
Removed the tracing and tested just the RPCs.